enterprise/backend/src/metabase_enterprise/serialization/cmd.clj

```diff
@@ -181,21 +183,43 @@
 
 (defn v2-dump
   "Exports Metabase app data to directory at path"
-  [path {:keys [user-email collections] :as opts}]
+  [path {:keys [user-email collection-ids] :as opts}]
   (log/info (trs "Exporting Metabase to {0}" path) (u/emoji "🏭 🚛💨"))
   (mdb/setup-db!)
   (t2/select User) ;; TODO -- why??? [editor's note: this comment originally from Cam]
   (serdes/with-cache
     (-> (cond-> opts
-         (seq collections) (assoc :targets (v2.extract/make-targets-of-type "Collection" collections))
-         user-email        (assoc :user-id (t2/select-one-pk User :email user-email :is_superuser true)))
+          (seq collection-ids) (assoc :targets (v2.extract/make-targets-of-type "Collection" collection-ids))
+          user-email        (assoc :user-id (t2/select-one-pk User :email user-email :is_superuser true)))
         v2.extract/extract
         (v2.storage/store! path)))
-  (log/info (trs "Export to {0} complete!" path) (u/emoji "🚛💨 📦")))
+  (log/info (trs "Export to {0} complete!" path) (u/emoji "🚛💨 📦"))
+  ::v2-dump-complete)
 
 (defn seed-entity-ids
   "Add entity IDs for instances of serializable models that don't already have them.
 
   Returns truthy if all entity IDs were added successfully, or falsey if any errors were encountered."
   []
   (v2.seed-entity-ids/seed-entity-ids!))
+
+
+(comment
```

Comment:
Yeah, I figured we could leave it here until we move it to a script or part of the release code.

Reply:
> I tried running your branch (with an existing app db) but don't see the Instance Analytics collection. Anything I could be doing wrong?

I'll update the PR, but this should only load the instance analytics if there is a directory named `instance_analytics` in the resources. I renamed the test data in test-resources to `instance_analytics_skip` so that it would not load. Then in the tests I point the loader to the `instance_analytics_skip` resource dir to make sure it can load something.

Reply:
Done.